.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ executors:
   # should also be updated.
   golang:
     docker:
-      - image: circleci/golang:1.17
+      - image: cimg/go:1.18
```

Comment:
shouldn't we use the image "quay.io/prometheus/golang-builder:1.18-base" ?

Reply:
No, we only use that image for `prometheus/prometheus` due to the frontend build needs. For all of the exporters we use CircleCI's image.